tests/tck-build-logic/src/main/groovy/org/graalvm/internal/tck/harness/tasks/TestInvocationTask.groovy

```diff
@@ -13,9 +13,9 @@ import org.gradle.api.tasks.Input
 import javax.inject.Inject
 import java.nio.file.Path
 import java.util.stream.Collectors
+import static org.graalvm.internal.tck.Utils.splitCoordinates;
+import static org.graalvm.internal.tck.Utils.readIndexFile;
 
-import static org.graalvm.internal.tck.Utils.readIndexFile
-import static org.graalvm.internal.tck.Utils.splitCoordinates
 /**
```

Comment:
revert?

Reply:
I accidentally deleted those imports and returned them in some of the next commits.